service/index/reader.go

```diff
@@ -16,25 +15,36 @@ import (
 )
 
 // ConcurrentPathReadLimit sets the number of concurrent paths in the Values lookup
-const ConcurrentPathReadLimit int = 128
+const ConcurrentPathReadLimit int = 1024
```

Comment:
this seems like a premature optimization.  cadence don't do batch read ...

Reply:
Yep, have a followup fix for the fast path: https://github.com/onflow/flow-archive/pull/120